app/views/complaints/show.html.erb

```diff
@@ -78,53 +78,22 @@
   </div>
 </div>
 
-<%# MODAL %>
+<%= render partial: "complaint_unlink_modal", locals: {
+  item: "tta",
```

Comment:
If we don't mind changing some of the language in the modal, I think we could remove some of the parameters feeding in, but I was trying to change as little as possible on the frontend of things